lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/TelemetryAppender.java

```diff
@@ -0,0 +1,35 @@
+package org.enso.logging.service.logback;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import java.net.URI;
+import java.util.ServiceLoader;
+import java.util.concurrent.ThreadPoolExecutor;
+
+public abstract class TelemetryAppender extends AppenderBase<ILoggingEvent> {
+
+  static TelemetryAppender load() {
+    var loader =
+        ServiceLoader.load(TelemetryAppender.class, TelemetryAppender.class.getClassLoader());
+    var appender = loader.findFirst().get();
+    return appender;
```

Comment:
[Injectable singletons](http://wiki.apidesign.org/wiki/Injectable_Singleton) are great, aren't they? I am always happy to see their usage - it increases **loosely coupling** of various components and improves modularity.

Just a warning: this is not a singleton: `load()` always creates new instance.

Reply:
> Just a warning: this is not a singleton: load() always creates new instance.

That should be fine. This method is called just once from [LogbackSetup.setupTelemetryAppender](https://github.com/enso-org/enso/blob/92ecb4a0f4e0d6d80d7c9a417834f2a85767949f/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/LogbackSetup.java#L318). Moreover, [TelemetryAppenderImpl](https://github.com/enso-org/enso/blob/05471a82b66cf9ac6b666b6a956d827098c5fbba/lib/java/logging-service-telemetry/src/main/java/org/enso/logging/service/telemetry/TelemetryAppenderImpl.java) is not restricted to be a singleton. But it does not make a lot of sense to have multiple instances of that class at the same time.